powershelf-manager/internal/proto/v1/powershelf-manager.proto

```
@@ -28,9 +28,9 @@ service PowershelfManager {
 
     // Power Control
     // Power OFF the rack
-    rpc PowerOff(PowershelfRequest) returns (PowerControlResponse);
+    rpc PowerOff(PowerRequest) returns (PowerControlResponse);
```

Buf (1.67.0)

[error] "v1.PowerControlResponse" is used as the request or response type for multiple RPCs.

(RPC_REQUEST_RESPONSE_UNIQUE)